g2p_registry_membership/models/group_membership.py

```diff
@@ -116,6 +117,15 @@ def _name_search(
             args = [("group", operator, name)] + args
         return self._search(args, limit=limit, access_rights_uid=name_get_uid)
 
+    @api.depends("ended_date")
+    def _compute_is_ended(self):
+        for rec in self:
+            is_ended = False
+            if rec.ended_date:
+                is_ended = True
+
+            rec.is_ended = is_ended
+
```

Comment:
Should we then add a constraint to make sure the ended_date is not in the future?

Reply:
I added the checking of ended_date @jeremi. Please check again.